toolkit/tools/pkg/isomakerlib/isomaker.go

```diff
@@ -101,25 +108,51 @@ func NewIsoMakerWithConfig(unattendedInstall, enableBiosBoot bool, baseDirPath,
 		outputDirPath:      outputDir,
 		imageNameBase:      imageNameBase,
 		imageNameTag:       imageNameTag,
-	}
+	}, nil
 }
 
 // Make builds the ISO image to 'buildDirPath' with the packages included in the config JSON.
-func (im *IsoMaker) Make() {
-	defer im.isoMakerCleanUp()
+func (im *IsoMaker) Make() (err error) {
+	defer func() {
+		cleanupErr := im.isoMakerCleanUp()
+		if cleanupErr != nil {
+			if err != nil {
+				err = fmt.Errorf("%w\n%w", err, cleanupErr)
```

Comment:
It might be worth to distinguish between the two errors here to make debugging easier.

Reply:
I had `%w\nError:%w` before, but opted to simplify...
I find it useful to distinguish between clean-up errors and real errors that affect the output.
Would something like `%w:\nclean-up error:%w` be okay?
@cwize1 @PawelWMS

Reply:
added `clean-up error:` prefix to error messages from clean-up...